citi-onboarding/Recorda-one-page
# Feature/navbar

### What I did:
- Added some images to "assets" folder
- Created the "Navbar" page
- Exported the "Navbar" to the react app

### How to test:

- git checkout main
- git pull
- git checkout feature/navbar
- write "npm start" on the client terminal
- check the navbar
- review the code
